LLVM brainfuck example can't compile on Arch Linux:  undefined reference to `llvm_ext.cc.c4a0b00d'

## Bug Report

Code for reproduce: [here](https://github.com/crystal-lang/crystal/blob/97e9b3dfa2c6fe0325c837633a6189be99e79eb9/samples/llvm/brainfuck.cr). 

1. download branfuck.cr and store it as brainfuck.cr
2. `$ crystal brainfuck.cr`
3. got error:
```
/usr/bin/ld: /tmp/cc8FuTGM.ltrans0.ltrans.o:(.debug_info+0x2b): undefined reference to `llvm_ext.cc.c4a0b00d'
/usr/bin/ld: /tmp/cc8FuTGM.ltrans0.ltrans.o:(.debug_info+0x34): undefined reference to `llvm_ext.cc.c4a0b00d'
/usr/bin/ld: /tmp/cc8FuTGM.ltrans0.ltrans.o:(.debug_info+0x4f): undefined reference to `llvm_ext.cc.c4a0b00d'
/usr/bin/ld: /tmp/cc8FuTGM.ltrans0.ltrans.o:(.debug_info+0x56): undefined reference to `llvm_ext.cc.c4a0b00d'
/usr/bin/ld: /tmp/cc8FuTGM.ltrans0.ltrans.o:(.debug_info+0x5e): undefined reference to `llvm_ext.cc.c4a0b00d'
/usr/bin/ld: /tmp/cc8FuTGM.ltrans0.ltrans.o:(.debug_info+0x79): more undefined references to `llvm_ext.cc.c4a0b00d' follow
collect2: error: ld returned 1 exit status
Error: execution of command failed with exit status 1: cc "${@}" -o /home/lyh/.cache/crystal/crystal-run-hello.tmp  -rdynamic -L/usr/bin/../lib/crystal /usr/lib/crystal/llvm/ext/llvm_ext.o `"/usr/bin/llvm-config" --libs --system-libs --ldflags 2> /dev/null` -lstdc++ -lpcre2-8 -lm -lgc -lpthread -levent  -lrt -lpthread -ldl
```

Crystal Version: `Crystal 1.8.2 (2023-05-11) LLVM: 15.0.7 Default target: x86_64-pc-linux-gnu`
Linux Version: `Linux arch 6.3.7-arch1-1 #1 SMP PREEMPT_DYNAMIC Sat, 10 Jun 2023 00:35:35 +0000 x86_64 GNU/Linux`
LLVM Version: `15.0.7`

--- a/src/llvm/builder.cr
+++ b/src/llvm/builder.cr
@@ -89,12 +89,24 @@ class LLVM::Builder
     Value.new LibLLVMExt.build_call2(self, func.function_type, func, (args.to_unsafe.as(LibLLVM::ValueRef*)), args.size, bundle, name)
   end
 
-  def call(type : LLVM::Type, func : LLVM::Function, args : Array(LLVM::Value), name : String = "", bundle : LLVM::OperandBundleDef = LLVM::OperandBundleDef.null)
+  def call(type : LLVM::Type, func : LLVM::Function, args : Array(LLVM::Value), name : String = "")
+    # check_type("call", type)
+    # check_func(func)
+    # check_values(args)
+
+    Value.new LibLLVM.build_call2(self, type, func, (args.to_unsafe.as(LibLLVM::ValueRef*)), args.size, name)
+  end
+
+  def call(type : LLVM::Type, func : LLVM::Function, args : Array(LLVM::Value), name : String, bundle : LLVM::OperandBundleDef)
     # check_type("call", type)
     # check_func(func)
     # check_values(args)
 
     Value.new LibLLVMExt.build_call2(self, type, func, (args.to_unsafe.as(LibLLVM::ValueRef*)), args.size, bundle, name)
+  end
+
+  def call(type : LLVM::Type, func : LLVM::Function, args : Array(LLVM::Value), bundle : LLVM::OperandBundleDef)
+    call(type, func, args, "", bundle)
   end
 
   def alloca(type, name = "")
@@ -281,7 +293,14 @@ class LLVM::Builder
     Value.new LibLLVMExt.build_invoke2 self, fn.function_type, fn, (args.to_unsafe.as(LibLLVM::ValueRef*)), args.size, a_then, a_catch, bundle, name
   end
 
-  def invoke(type : LLVM::Type, fn : LLVM::Function, args : Array(LLVM::Value), a_then, a_catch, bundle : LLVM::OperandBundleDef = LLVM::OperandBundleDef.null, name = "")
+  def invoke(type : LLVM::Type, fn : LLVM::Function, args : Array(LLVM::Value), a_then, a_catch, *, name = "")
+    # check_type("invoke", type)
+    # check_func(fn)
+
+    Value.new LibLLVM.build_invoke2 self, type, fn, (args.to_unsafe.as(LibLLVM::ValueRef*)), args.size, a_then, a_catch, name
+  end
+
+  def invoke(type : LLVM::Type, fn : LLVM::Function, args : Array(LLVM::Value), a_then, a_catch, bundle : LLVM::OperandBundleDef, name = "")
     # check_type("invoke", type)
     # check_func(fn)
 

